### ISSUE TITLE
13281264: Replace UDID limits for ad-hoc builds with Apple IDs

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#5251]
#### Description

Currently, ad-hoc provisioning profiles are tied to specific UDIDs for up to 100 devices. I understand why there's a limit and why it's refreshable only once a year, but it's still way more limiting then it needs to be. Especially now, with the expansion of iOS device portfolio, I feel it has become a serious obstacle.

Proposal:
- AppleIDs are associated with ad-hoc provisioning profile instead of UDIDs
- AppleIDs must be iCloud-enabled accounts or iOS developer accounts
- A new App Store like app will list the app builds that particular iCloud account has access to
- Installation on the device will work the same way as from the App Store, iOS will be able to check the signing, maybe ask the tester to sign into iCloud and install the app
- This new app will notify user about beta updates, as App Store app does now

Some details:
- Developers are responsible to collect AppleIDs and put them into Prov. Portal
- End-user accounts will have to confirm that they want to be associated with particular developer
- Testers can be associated with any number of developer accounts
- Ideally, provisioning portal will offer creation of specific invitation link that developers will send their testers, where they just have to tap Yes and be part of the team. Their account will then show up as recruited

Nice to have:
- Developer can create free redeem codes for his testers, which are independent of the existing 50 promo codes
- Instead of crude total limit of test accounts per developer, each app being tested will have a maximum of 100 Apple IDs associated with it.

This is just the essence, more details would be easily worked out if you find this.
## 

Product Version: 
Created: 2013-02-24T10:34:16.593377
Originated: 2013-02-24T10:33:12+00:00
Open Radar Link: http://www.openradar.me/13281264
